src/frontend/js/components/PaginateCourseSearch/index.tsx

```diff
@@ -123,15 +125,20 @@ export const PaginateCourseSearch = ({ courseSearchTotalCount }: PaginateCourseS
                 </li>
               ) : (
                 <li className="pagination__item">
-                  <button
+                  <a
+                    href={`?${stringify({
+                      ...parse(location.search),
+                      offset: String((page - 1) * limit),
+                    })}`}
                     className="pagination__page-number"
-                    onClick={() =>
+                    onClick={(event) => {
+                      event.preventDefault();
                       dispatchCourseSearchParamsUpdate({
                         // Pages are 1-indexed, we need to 0-index them to calculate the correct offset
                         offset: String((page - 1) * limit),
                         type: CourseSearchParamsAction.pageChange,
-                      })
-                    }
```

Comment:
I wonder if it's not overkill but we could keep the link default behavior when user presses shortcut keys.

We could do nothing if user press ctrl, shift or meta key to open link in a new tab or window.

Ctrl + Click : Open a new tab on Window (On Mac OS, it behaves like a right click so click event is not dispatched)
Meta + Click : Open a new tab on Mac OS
Shift + Click : Open a new window on Mac OS and Window 

I don't know the behaviour on Linux, so may be my suggestion becomes too complicated to support every cases.

```
onClick={(event) => {
            if(!event.metaKey && !event.ctrlKey && !event.shiftKey) {
                      event.preventDefault();
                      dispatchCourseSearchParamsUpdate({
                        // Pages are 1-indexed, we need to 0-index them to calculate the correct offset
                        offset: String((page - 1) * limit),
                        type: CourseSearchParamsAction.pageChange,
                      })
                    }
            }
```

Reply:
Oh sure :)